packages/block-library/src/spacer/edit.js

```diff
@@ -87,13 +87,56 @@ const SpacerEdit = ( {
 	context,
 	__unstableParentLayout: parentLayout,
 	className,
+	clientId,
 } ) => {
 	const disableCustomSpacingSizes = useSelect( ( select ) => {
 		const editorSettings = select( blockEditorStore ).getSettings();
 		return editorSettings?.disableCustomSpacingSizes;
 	} );
-	const { orientation } = context;
+
+	const galleryChildBlocks = useSelect( ( select ) => {
```

Comment:
I wonder if we need to make this gallery specific. This could potentially be useful in other column related layouts at some point so maybe this could be `columnSiblingBlocks` and ` isInsideGallery` could be ` isInsideColumns` ?

Reply:
Good idea! Do you have any other cases already in mind?

Reply:
I don't have any specific use cases in mind, but it opens the possibility for more options if we make it more generic up front.